Add examples of recursion (#12833)
mdn/content: files/en-us/glossary/recursion/index.md

2
1

### Recursion is limited by stack size

> The following JavaScript code defines a function that returns a number of max call stack size available for it in JavaScript runtime.

Reviewer: ```suggestion
The following code defines a function that returns the maximum call-stack size available in the JavaScript runtime in which the code is run.
```

> The following code defines a function that returns the maximum call-stack size available in the JavaScript runtime in which the code is run.

Reviewer: ```suggestion
The following code defines a function that returns the maximum size of the call stack available in the JavaScript runtime in which the code is run.
```
